EngineLayer/PsmTsv/PsmFromTsv.cs

```diff
@@ -38,13 +40,16 @@ public class PsmFromTsv
         public string ProteinName { get; }
         public string GeneName { get; }
         public string OrganismName { get; }
+        public string IdentifiedSequenceVariations { get; }
         public string PeptideDesicription { get; }
         public string StartAndEndResiduesInProtein { get; }
         public string PreviousAminoAcid { get; }
         public string NextAminoAcid { get; }
         public string DecoyContamTarget { get; }
         public double? QValueNotch { get; }
 
+        public Dictionary<MatchedFragmentIon, bool> VariantCrossingIons { get; }
```

Comment:
The dictionary should be renamed it is confusing because it seems that this dictionary would only contains fragment ions that cross the variant, which is not the case.

Reply:
Is that possible to add a property to MatchedFragmentIon, like "bool ionCrossingVariant {get; set;}" ? So no extra dictionary is needed here.

Reply:
That was the original plan but Rob wanted to avoid making changes to mzlib, which that required.